app/client/views/application/application.html

```diff
@@ -198,7 +198,7 @@
 
         <div class="field">
           <label>
-            Resume (PDF)
+            Resume  (PDF less than 0.5 MB)
```

Comment:
```suggestion
            Resume (PDF less than 0.5 MB)
```

Reply:
what about `Resume (PDF, max 0.5 MB)?

Reply:
Yea I think I like that better. @kbuci Go with Chris' comment. But still remove that extra space.

Reply:
```suggestion
            Resume (PDF, 0.5 MB max)
```